doc/api_assets/style.css

```diff
@@ -360,7 +372,7 @@ span.type {
   margin-top: 1.25em;
   background: #f2f5f0;
   margin-bottom: 0;
-  padding-bottom: 2.5em;
+  padding-bottom: 5em;
```

Comment:
Unfortunately it needs to be big now with the gradient, or else items would still be faded out when scrolled to the end. I think 4em is the lower limit, I'll experiment a bit.